static/src/javascripts/projects/common/modules/check-mediator.js

```diff
@@ -90,17 +51,16 @@ const waitForCheck = (id: string): Promise<any> => {
     return Promise.reject(`No deferred check with id ${id}`);
 };
 
+const initCheckMediator = (): void => {
+    checks.forEach(registerCheck);
+};
```

Comment:
If this function initially set `registeredChecks = {}`, we wouldn't need to expose the `testClean` function 🎉